src/coreclr/src/vm/eetwain.cpp

```diff
@@ -3079,42 +3128,54 @@ unsigned SKIP_ALLOC_FRAME(int size, PTR_CBYTE base, unsigned offset)
         return (SKIP_PUSH_REG(base, offset));
     }
 
-    if (size >= (int)GetOsPageSize())
+    const int pageSize = (int)GetOsPageSize();
```

Comment:
Should this be hard-coded number instead of the OS page size to avoid problem with OS page size changing between AOT time and JIT time?

Reply:
Yes, it should. Thanks for catching this.

Reply:
Hmm, if I take an approach as discussed in another PR - that this should match target page size for JIT-ed code and `0x1000` for R2R code, right?

Reply:
The runtime assumes that the unwinding algorithm is same for all code flavors.